src/components/dev-hub/stories/badge.stories.mdx

```diff
@@ -0,0 +1,53 @@
+import { Meta, Story, Preview } from '@storybook/addon-docs/blocks';
+import Badge from '../badge';
+import { colorMap } from '../theme';
+
+<Meta title="Badge" />
+
+# Badge
+
+Badges can be used to label contents. It accepts two parameters: `contentType` and `color`.
+* The `contentType` variable defines the text that goes inside the badge.
+* The optional `color` variable defines the color of the borders around the badge.
```

Comment:
Can we add a note on how it is `position: absolute` by default? Definitely good to know in case someone wants to use this otherwise.